FrameStreamSockInput.go

```diff
@@ -49,7 +55,7 @@ func (input *FrameStreamSockInput) ReadInto(output chan []byte) {
 			log.Printf("net.Listener.Accept() failed: %s\n", err)
 			continue
 		}
-		i, err := NewFrameStreamInput(conn, true)
+		i, err := NewFrameStreamInputTimeout(conn, true, input.timeout)
```

Comment:
The bi=false disabling timeouts is best documented in golang-framestream, where it should be explicit that the I/O timeouts are only relevant for bidirectional (connection-based) encoders and decoders.

The "bidirectional" naming refers only to the control traffic of the Frame Streams connection; the data always flows from Encoder to Decoder. A "bidirectional" one of either merely sends feedback to the other, where the "unidirectional" version does not require the active participation of the peer -- just sending ore reading a series of bytes. As such, the *De*coder vs. *En*coder is the only thing semantically significant to the caller; the bidirectionality is all about what goes on behind the scenes.

Reply:
Yes, I know this now - just thought it would save the reader some time if we documented this somewhere (e.g. "bidirectional" means "connection-oriented" like TCP, "unidirectional" means "connection-less" like UDP). Anyway, feel free to skip.